pyevmasm/evmasm.py

```diff
@@ -355,13 +364,25 @@ def assemble_one(asmcode, pc=0, fork=DEFAULT_FORK):
             instr.pc = pc
         if instr.operand_size > 0:
             assert len(asmcode) == 2
-            instr.operand = int(asmcode[1], 0)
+            operand = asmcode[1].strip()
+            if is_push(instr) and not is_digit(operand):
+                # instantiating a label, fill it with zeros instead
+                instr.operand = 0
+                if operand in fillins:
+                    fillins[operand].append(pc)
+                else:
+                    fillins[operand] = [pc]
+            else:
+                instr.operand = int(asmcode[1], 0)
```

Comment:
Idea:

Not sure we need the fillings dict? 
Maybe just write whatever there is to the instruction.operand. An int or label(string).
And then consider the instructions with a string operand to need fixup. ?

Instruction will not be able to generate a bytecode until it is "fixedup"